indexers/gemini-3h/leaderboard/project.ts

```diff
@@ -8,6 +8,7 @@ import path from "path";
 // Load the appropriate .env file
 const dotenvPath = path.resolve(__dirname, `../../../.env`);
 dotenv.config({ path: dotenvPath });
+console.log("endpoint", process.env.GEMINI_3H_RPC);
```

Comment:
Are this `console.logs` intended or were only for debugging?

Reply:
I created an issue about this https://github.com/autonomys/astral/issues/883 I'll merge as is since this is not a blocker